src/test/java/pages/administrator/AdministratorPage.java

```diff
@@ -22,15 +30,26 @@ public HeaderGeneralPageComponent getHeaderGeneralPageComponent() {
         return new HeaderGeneralPageComponent(driver);
     }
 
-    public TabPanelPage getTabPanelPage() {
-        return new TabPanelPage(tabDiv);
+    public TabPanelComponent getTabPanelPage() {
+        return new TabPanelComponent(tabDiv);
+    }
+
+    public void setOrderId(String orderId){
```

Comment:
Do you really need this method? `this.orderId ` is never used, so I don't see the reasons to have it in the class.

Reply:
Right, just forgot to remove. Will do.